plugins/modules/activation_key.py

```diff
@@ -82,7 +82,9 @@
   content_overrides:
     description:
       - List of content overrides that include label and override state
-      - Label refers to repository C(content_label), e.g. rhel-7-server-rpms
+      - Label refers to repository C(content_label).
+      - For Red Hat products for example C(rhel-7-server-rpms).
+      - For custom products it's in the format: C(<organization_label>_<product_label>_<repositoryLabel>), e.g. C(ExampleOrg_ExampleProduct_ExampleRepository).
```

Comment:
```suggestion
      - For custom products it's in the format C(<organization_label>_<product_label>_<repositoryLabel>), e.g. C(ExampleOrg_ExampleProduct_ExampleRepository).
```

The colon confuses YAML. Either we need to quote the line, or drop the colon, and in this case I think w/o the colon it's better readable.